src/main/java/org/folio/marc4ld/service/marc2ld/mapper/mapper/MapperUtils.java

```diff
@@ -0,0 +1,58 @@
+package org.folio.marc4ld.service.marc2ld.mapper.mapper;
+
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import lombok.RequiredArgsConstructor;
+import org.folio.ld.dictionary.model.Resource;
+import org.marc4j.marc.ControlField;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class MapperUtils {
```

Comment:
Since it's not an util class anymore, we should also name it differently, right?
What about MapperHelper, CommonMapper, CoreMapper, etc ?